### PR TITLE
 Implemented some operations in the webservice authorization.

### PR DESCRIPTION
**Pull request by cristianopacheco**
* LoginController renamed to AuthController

* The authentication route was changed to /auth/issue (suggestion @hernandev ), this change was reflected in the client.

* Added method to revoke token

* Added method to update token

* The authentication middleware now used is the jwt library,
Because with the standard laravel middleware, it is not possible to identify when the token is expired.

I talked to @vedovelli  about the refresh token feature on the client, so we will open a discussion in this PR on how to implement this feature.

I will give an example of how the api returns the response signaling that the token is expired.

### CODE REVIEW

**Comment by vedovelli** [comment]
OK @cristianopacheco so I run your branch locally and everything is working fine.

What's your question regarding token refreshing?

File: webservice/app/Http/Controllers/AuthController.php [hunk 1]
```diff
@@ -93,6 +92,31 @@ public function sendLockoutResponse(Request $request)
...
+     * Refresh the user's token.
+     *
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function refreshToken(Request $request)
+    {
+        $token = Auth::guard('api')->refresh();
+
+        return $this->response(compact('token'));
```
Comment (vedovelli) [inline webservice/app/Http/Controllers/AuthController.php]: What is gonna be returned when token cannot be refreshed? I understand there's a constraint of how many times a token can be refreshed.

Comment (cristianopacheco) [inline webservice/app/Http/Controllers/AuthController.php]: Will be returned a 500 error by the jwt lib and the user will be redirected to login page. The token can be updated as many times as needed for 2 weeks (default setting).

**Comment by vinicius73** [comment]
i will accept this PR and make a implementation for the proposed problem

**Comment by vinicius73** [comment]
@cristianopacheco if is possible, make a issue with this question.

**Comment by vedovelli** [comment]
This PR was reverted because it contains non working code.

**Pull request merged**